events/wmt20.md

```diff
@@ -217,3 +217,46 @@ Friday, November 20, 2020
 | | [***The Tatoeba Translation Challenge – Realistic Data Sets for Low Resource and Multilingual MT***](https://aclanthology.org/2020.wmt-1.139.pdf) <br>Jörg Tiedemann |
 | | [***Human-Paraphrased References Improve Neural Machine Translation***](https://aclanthology.org/2020.wmt-1.140.pdf) <br>Markus Freitag, George Foster, David Grangier and Colin Cherry |
 | |	[***Incorporating Terminology Constraints in Automatic Post-Editing***](https://aclanthology.org/2020.wmt-1.141.pdf) <br>David Wan, Chris Kedzie, Faisal Ladhak, Marine Carpuat and Kathleen McKeown |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2020 Conference on Machine Translation (WMT20)***](https://aclanthology.org/2020.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English
+
+| Language pair | Annotation style | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- | --- |
+| Chinese→English | Segment rating with document context | 77.5 | 0.102 | `VolcTrans` |
+| Czech→English | Segment rating with document context | 78.3 | 0.118 | `CUNI-DocTransformer` |
+| German→English | Segment rating with document context | 82.6 | 0.228 | `VolcTrans` |
+| Inuktitut→English | Segment rating without document context | 73.1 | 0.168 | `NiuTrans` |
+| Japanese→English | Segment rating with document context | 75.1 | 0.184 | `Tohoku-AIP-NTT` |
+| Khmer→English | Segment rating without document context | 69.0 | 0.168 | `Online-B` |
+| Pashto→English | Segment rating without document context | 67.3 | 0.032 | `Online-B` |
+| Polish→English | Segment rating with document context | 77.2 | 0.131 | `SRPOL` |
+| Russian→English | Segment rating with document context | 79.3 | 0.124 | `Online-G` |
+| Tamil→English | Segment rating with document context | 68.7 | 0.203 | `GTCOM` |
+
+#### English →
+
+| Language pair | Annotation style | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- | --- |
+| English→Chinese | Segment rating with document context | 80.6 | 0.568 | `HUMAN-B` |
+| English→Czech | Segment rating with document context | 85.6 | 0.654 | `HUMAN` |
+| English→German | Segment rating with document context | 90.5 | 0.569 | `HUMAN-B` |
+| English→Inuktitut (News only) | Segment rating without document context | 90.5 | 0.574 | `HUMAN` |
+| English→Japanese | Segment rating with document context | 79.7 | 0.576 | `HUMAN` |
+| English→Polish | Segment rating with document context | 88.6 | 0.672 | `HUMAN` |
+| English→Russian | Segment rating with document context | 91.8 | 0.681 | `HUMAN` |
+| English→Tamil | Segment rating with document context | 83.4 | 0.762 | `HUMAN` |
+| English→Khmer | Segment rating without document context | 77.4 | 0.478 | `GTCOM` |
+| English→Pashto | Segment rating without document context | 73.0 | 0.244 | `GTCOM` |
+
+#### French to/from German (segment rating without document context)
```

Comment:
Hi @liashahnazaryan!

Maybe to keep consistency with the previous sub-headers, this header should be spelled with the arrow characters as well.

I would personally use `French <> German` because, as translators, we always list our language pairs with the `<>` characters (or just `>`, as in `English > Spanish`).  This may be different for researchers or academia.
Otherwise, we can just use `Into English`, `From English`, and `French into/from German`.

What do you think?

Reply:
Agree. I first wanted to use the left-right arrow, but it converts to an emoji, so I chose to go ahead with the `from/to` option. But `<>` seems a good substitute for all cases, and I guess it also applies to WMT more than the other options as they also use arrows instead of `from/to`.

Reply:
> it converts to an emoji

Maybe you need to escape it, like this `\>`?

Reply:
the problem is solved now I guess, you can see the changes in the updated file
I'll now apply the changes to the wmt21 results as well